### ISSUE TITLE
VxMark: Write-in candidate modal no longer reads out the intro text/audio instructions (at least on current `main` on my VM)

### ISSUE THREAD

**Issue by arsalansufi** [votingworks/vxsuite#6528]
*VxMark:* Write-in candidate modal no longer reads out the intro text/audio instructions (at least on current `main` on my VM)

Issue automatically created from Slack feedback reported by kofi: 
https://votingworks.slack.com/archives/CJU9MSC6S/p1746734620672839
